drivers/soundwire/debugfs.c

```diff
@@ -118,6 +118,32 @@ static int sdw_slave_reg_show(struct seq_file *s_file, void *data)
 }
 DEFINE_SHOW_ATTRIBUTE(sdw_slave_reg);
 
+#define sdw_slave_transaction_attr(field)				\
+static int field##_show(struct seq_file *s_file, void *data) 		\
+{									\
+	struct sdw_slave *slave = s_file->private;			\
+	char *buf;							\
+	ssize_t ret;							\
+									\
+	buf = kzalloc(RD_BUF, GFP_KERNEL);				\
```

Comment:
RD_BUF = 3*PAGE_SIZE(4KB), do we need this size of buffer to show a integer. Or a simple buf[8] to avoid kzalloc and kfree.

Reply:
yes, this was a copy-paste from the registers case, will fix. thanks for the suggestion @RanderWang